cmd/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 	kClient := ik.NewClient(specs.KratosPublicURL)
 	hClient := ih.NewClient(specs.HydraAdminURL)
 
-	router := web.NewRouter(kClient, hClient, distFS, tracer, monitor, logger)
+	router := web.NewRouter(kClient, hClient, distFS, tracer, monitor, logger, specs.BaseURL)
```

Comment:
```suggestion
	router := web.NewRouter(specs.BaseURL, kClient, hClient, distFS, tracer, monitor, logger)
```

as a rule of thumb, let s try to keep the "auxiliary" parts at the end, aka, `tracer`, `monitor` and `logger`